src/query/ee/tests/it/aggregating_index/index_scan.rs

```diff
@@ -64,8 +64,8 @@ async fn test_index_scan_agg_args_are_expression() -> Result<()> {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn test_fuzz() -> Result<()> {
-    test_fuzz_impl("parquet").await?;
-    test_fuzz_impl("native").await
+    test_fuzz_impl("parquet").await
+    // test_fuzz_impl("native").await
```

Comment:
Doesn't the native format need to be tested?

Reply:
Oh, I forgot to uncomment this. Sry